app/controllers/my_facilities_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+class MyFacilitiesController < AdminController
+  include DistrictFiltering
+  include Pagination
+  include SizeFiltering
+
+  before_action :authorize_my_facilities, only: %i[index ranked_facilities blood_pressure_control]
+
+  def index
+    @users_requesting_approval = policy_scope([:manage, :user, User])
+                                 .requested_sync_approval
+                                 .order(updated_at: :desc)
+
+    @users_requesting_approval = paginate(@users_requesting_approval)
+
+    @facilities = policy_scope([:manage, :facility, Facility])
+    @inactive_facilities = MyFacilitiesQuery.inactive_facilities(@facilities)
+
+    @facility_counts_by_size = { total: @facilities.group(:facility_size).count,
+                                 inactive: @inactive_facilities.group(:facility_size).count }
+
+    @bp_counts = { last_week: @inactive_facilities.bp_counts_in_period(1.week.ago, Time.current),
+                   last_month: @inactive_facilities.bp_counts_in_period(1.month.ago, Time.current) }
+  end
+
+  def ranked_facilities; end
+
+  def blood_pressure_control; end
+
+  private
+
+  def authorize_my_facilities
+    authorize(:dashboard, :show?)
```

Comment:
Should this be `authorize(:dashboard, :view_my_facilities?)` to make use of [this bit](https://github.com/simpledotorg/simple-server/pull/710/files#diff-5182f4f26eef1925711f3c6a19e91fcbR45-R47)?

Reply:
We could do this later. For now, we want people who navigate to '/my_facilities' to be able to see the page. We're using the :view_my_facilities permission to hide the link in the layout.